package.json

```diff
@@ -15,7 +15,7 @@
   },
   "devDependencies": {
     "@babel/cli": "7.22.10",
-    "@babel/core": "7.22.11",
+    "@babel/core": "7.23.6",
     "@babel/plugin-proposal-nullish-coalescing-operator": "7.18.6",
```

Comment:
Gjerne bytt ut denne og den under. De er begge deprecated.

Reply:
Mener det er transform i stedet for proposal 😎

Reply:
done 👌